src/it/resources-noncompilable/com/google/checkstyle/test/chapter5naming/rule527localvariablenames/InputPatternVariableNameEnhancedInstanceofTestDefault.java

```diff
@@ -0,0 +1,83 @@
+//non-compiled with javac: Compilable with Java14
+package com.google.checkstyle.test.chapter5naming.rule527localvariablenames;
+
+import java.util.ArrayList;
+import java.util.Locale;
+/*
+ * Config:
+ * tokens = PATTERN_VARIABLE_DEF
+ * format = "^[a-z][a-zA-Z0-9]*$"
```

Comment:
remove config from google inputs, it is not required as config is single.

We have rule for embedded config only for UTs, where we have numerous configs.
Google is single config, so copy of config will not do very good job, as google style is also changing periodically.

Reply:
Fixed.